functions/DatabaseRepository.fs

```diff
@@ -372,6 +376,62 @@ module DatabaseRepository =
             msg.Headers.Authorization <-  AuthenticationHeaderValue("Bearer", sharedSecret)
             sendAsync<seq<Person>> msg
 
+
+    let isServiceAdmin' netid = 
+        ["jhoerr"; "kendjone"; "jerussel"; "brrund"; "johndoe"] 
+        |> Seq.contains netid
+
+    let isServiceAdmin netid =  isServiceAdmin' netid |> ok
+
+    let hasCascadedUnitPermsSql = """
+    WITH RECURSIVE parentage AS (
+       -- first row
+       SELECT u.id, u.name, u.parent_id
+       FROM units u
+       WHERE u.id = @UnitId
+       UNION
+       -- recurse
+       SELECT u.id, u.name, u.parent_id
+       FROM units u
+       INNER JOIN parentage p ON p.parent_id = u.id
+    ) 
+    SELECT 
```

Comment:
```sql
    SELECT 
    	-- select 'true' if this person has the specified permissions
    	-- in this unit or any parent unit and 'false' otherwise.
    	CASE WHEN EXISTS (
    		SELECT pa.id, um.role, pe.netid
    		FROM parentage pa
    		JOIN unit_members um on pa.id = um.unit_id
    		JOIN people pe on pe.id = um.person_id 
    		WHERE
    			pe.netid = @NetId 
    			AND um.permissions = ANY(@UnitPermissions)
    	) 
    	THEN TRUE
    	ELSE FALSE
    	END
```
Could probably be:
```sql
    SELECT 
    	-- select 'true' if this person has the specified permissions
    	-- in this unit or any parent unit and 'false' otherwise.
        EXISTS (
            SELECT pa.id, um.role, pe.netid
    	    FROM parentage pa
    	    JOIN unit_members um on pa.id = um.unit_id
    	    JOIN people pe on pe.id = um.person_id 
    	    WHERE
    		pe.netid = @NetId 
    		AND um.permissions = ANY(@UnitPermissions)
    	)
```
Since `EXISTS()` returns a boolean already.

Reply:
Hah, good point! I'll make that change. Thanks!